libs/types/src/cdf/cast-vote-records/vx-schema.json

```diff
@@ -293,7 +293,7 @@
       "type": "object"
     },
     "CVR.CVRSnapshot": {
-      "required": ["@id", "@type", "Type"],
+      "required": ["@id", "@type", "Type", "CVRContest"],
```

Comment:
Rationale: every snapshot should have contests. If it doesn't, the CVR just an entirely blank sheet. In VVSG, undervotes should be represented explicitly by contest, so there will be a list of contests even if a voter does cast an entirely blank sheet.